include/exec/__detail/__bit_cast.hpp

```diff
@@ -37,9 +44,13 @@ namespace exec {
       std::is_trivially_copyable_v<_From>, "bit_cast requires _From to be trivially copyable");
     static_assert(
```

Comment:
Minor nit: the standard version has constraints for these requirements instead of mandates - perhaps so it's SFINAE-friendly?